docs/wallet/dlc.md

```diff
@@ -179,45 +168,15 @@ where the `eventid` is in all but the messages other than the DLC Offer message,
 
 ### Step 4: Executing the DLC
 
-#### Mutual Close
+#### Execute
 
-Upon receiving an oracle signature, either party can initiate a mutual close with
+Upon receiving an oracle signature, you can execute the DLC unilaterally with
 
 ```bashrc
-./app/cli/target/graalvm-native-image/bitcoin-s-cli initdlcmutualclose --eventid [eventid] --oraclesig [sig] --escaped
+./app/cli/target/graalvm-native-image/bitcoin-s-cli executedlc --eventid [eventid] --oraclesig [sig]
```

Comment:
So I think we should take out stuff about `graalvm-native-image` for now as you need to install extra stuff and me building it did not work. It would be simpler if he just uses the standard jar.